### PR TITLE
tools/lightning: update lightning docs

### PR DESCRIPTION
**Pull request by huangxiuyan**
Via: https://pingcap.com/docs/dev/reference/tools/tidb-lightning/monitor/ 
PTAL @dcalvin

### CODE REVIEW

File: tools/lightning/monitor.md [hunk 1]
```diff
@@ -113,6 +153,13 @@ scrape_configs:
     - **state**：`pending` / `written` / `closed` / `imported` / `altered_auto_inc` / `checksum` / `analyzed` / `completed`
     - **result**：`success` / `failure`
 
+**`lightning_engines`** (计数器)
+
+    计算处理后引擎文件的数量以及他们的状态。标签：
```
Comment (dcalvin) [inline tools/lightning/monitor.md]:
```suggestion
    计算处理后引擎文件的数量以及其状态。标签：
```

File: tools/lightning/monitor.md [hunk 2]
```diff
@@ -55,30 +55,70 @@ scrape_configs:
...
+
+    上传到 TiKV 节点的 SST 文件（压缩）大小的直方图。
+
+- **`tikv_import_sst_ingest_duration`** (直方图)
+
+    采集 SST 文件到 TiKV 需时的直方图。
+
+- **`tikv_import_each_phase`** (测量仪)
+
+    表示运行阶段。值为 1 表示在阶段内运行，值为 0 表示在阶段内运行。标签：
```
Comment (dcalvin) [inline tools/lightning/monitor.md]:
```suggestion
    表示运行阶段。值为 1 时表示在阶段内运行，值为 0 时表示在阶段外运行。标签：
```

File: tools/lightning/monitor.md [hunk 3]
```diff
@@ -55,30 +55,70 @@ scrape_configs:
...
+
+    从 `ImportSSTJob` 上传 SST 文件到 TiKV 节点需时的直方图。
+   
+- **`tikv_import_sst_chunk_bytes`** (直方图)
+
+    上传到 TiKV 节点的 SST 文件（压缩）大小的直方图。
+
+- **`tikv_import_sst_ingest_duration`** (直方图)
+
+    采集 SST 文件到 TiKV 需时的直方图。
```
Comment (dcalvin) [inline tools/lightning/monitor.md]:
```suggestion
    将 SST 文件传入至 TiKV 需时的直方图。
```

File: tools/lightning/monitor.md [hunk 4]
```diff
@@ -55,30 +55,70 @@ scrape_configs:
...
+    从 `tidb-lightning` 接收每个 KV 对区块需时直方图。
 
 - **`tikv_import_upload_chunk_bytes`**（直方图）
 
-    上传到 TiKV 的每个区块大小的直方图。
+     上传到 TiKV 的每个 SST 文件区块大小（压缩）的直方图。
+
+- **`tikv_import_range_delivery_duration`** (直方图)
+
+    将 KV 对中的一个 range 发送至 `dispatch-job` 需时的直方图。
```
Comment (dcalvin) [inline tools/lightning/monitor.md]:
```suggestion
    将一个 range 的 KV 对发送至 `dispatch-job` 需时的直方图。
```

File: tools/lightning/monitor.md [hunk 5]
```diff
@@ -55,30 +55,70 @@ scrape_configs:
...
+
+- **`tikv_import_split_sst_duration`** (直方图)
+
+    将 range 从引擎文件中分离到 SST 文件中需时的直方图。
```
Comment (dcalvin) [inline tools/lightning/monitor.md]:
```suggestion
    将 range 从引擎文件中分离到单个 SST 文件中需时的直方图。
```

File: tools/lightning/monitor.md [hunk 6]
```diff
@@ -55,30 +55,70 @@ scrape_configs:
...
+
+- **`tikv_import_sst_delivery_duration`** (直方图)
+
+    将 SST 文件从 `dispatch-job` 发送到 `ImportSSTJob` 需时的直方图
+    
+- **`tikv_import_sst_recv_duration`** (直方图)
+
+    `ImportSSTJob` 接收从 `dispatch-job` 发送过来的 SST 文件需时的直方图。
```
Comment (dcalvin) [inline tools/lightning/monitor.md]: @kennytm Please check the translation of this one. Actually, I am not sure about the original sentence in English. Note the **bold** part.

> Bucketed histogram of the time needed to receive an SST file from a `dispatch-job` **in an `ImportSSTJob`**.

File: tools/lightning/deployment.md [hunk 7]
```diff
@@ -44,7 +44,9 @@ category: tools
 
 > **注意：**
 >
-> `tidb-lightning` 是 CPU 密集型程序，如果和其它程序混合部署，需要通过 `region-concurrency` 限制 `tidb-lightning` 的 CPU 实际占用核数，否则会影响其他程序的正常运行。建议将混合部署机器上 75% 的 CPU 分配给 `tidb-lightning`。例如，机器为 32 核，则 `tidb-lightning` 的 `region-concurrency` 可设为 24。
+> - `tidb-lightning` 是 CPU 密集型程序，如果和其它程序混合部署，需要通过 `region-concurrency` 限制 `tidb-lightning` 的 CPU 实际占用核数，否则会影响其他程序的正常运行。建议将混合部署机器上 75% 的 CPU 分配给 `tidb-lightning`。例如，机器为 32 核，则 `tidb-lightning` 的 `region-concurrency` 可设为 24。
+>
+> - `tikv-importer` 将中间数据存储缓存到内存上以加速导入过程。占用内存大小可以通过 **(`max-open-engines` × `write-buffer-size` × 2) + (`num-import-jobs` × `region-split-size` × 2)** 计算得来。如果磁盘写入速度慢，缓冲区可能会带来更大的内存占用。
```
Comment (kennytm) [inline tools/lightning/deployment.md]: "缓冲区" → "缓存"

File: tools/lightning/monitor.md (hunk 6 above)
Comment (kennytm) [reply tools/lightning/monitor.md]: The `ImportSSTJob` and `dispatch-job` are both names of processes in Importer. We could add a Chinese description like "任务" after `ImportSSTJob` if we want to make this clearer.

**Comment by huangxiuyan** [comment]
@dcalvin @kennytm PTAL，thanks!

File: tools/lightning/monitor.md [hunk 8]
```diff
@@ -55,30 +55,70 @@ scrape_configs:
...
+
+    将 KV 对中的一个 range 发送至 `dispatch-job` 需时的直方图。
+
+- **`tikv_import_split_sst_duration`** (直方图)
+
+    将 range 从引擎文件中分离到 SST 文件中需时的直方图。
+
+- **`tikv_import_sst_delivery_duration`** (直方图)
+
+    将 SST 文件从 `dispatch-job` 发送到 `ImportSSTJob` 需时的直方图
```
Comment (dcalvin) [inline tools/lightning/monitor.md]:
```suggestion
    将 SST 文件从 `dispatch-job` 任务发送到 `ImportSSTJob` 任务需时的直方图
```

File: tools/lightning/monitor.md (hunk 6 above)
Comment (dcalvin) [inline tools/lightning/monitor.md]:
```suggestion
    `ImportSSTJob` 任务接收从 `dispatch-job` 任务发送过来的 SST 文件需时的直方图。
```

File: tools/lightning/deployment.md (hunk 7 above)
Comment (dcalvin) [inline tools/lightning/deployment.md]:
```suggestion
> - `tikv-importer` 将中间数据存储缓存到内存上以加速导入过程。占用内存大小可以通过 **(`max-open-engines` × `write-buffer-size` × 2) + (`num-import-jobs` × `region-split-size` × 2)** 计算得来。如果磁盘写入速度慢，缓存可能会带来更大的内存占用。
```

File: tools/lightning/monitor.md [hunk 9]
```diff
@@ -90,15 +130,15 @@ scrape_configs:
 
 - **`lightning_importer_engine`**（计数器）
 
-    计算已开启及关闭的引擎文档数量。标签：
+    计算已开启及关闭的引擎文件数量。标签：
 
     - **type**：`open` / `closed`
 
 - **`lightning_idle_workers`**（测量仪）
```
Comment (dcalvin) [inline tools/lightning/monitor.md]:
```suggestion
- **`lightning_idle_workers`**（计量表盘）
```

**Comment by lilin90** [comment]
@huangxiuyan The link in line 51 is incorrect.

**Pull request merged**